activities/service/v1/notifications.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+package io.spoud.sdm.activities.v1;
+
+option java_multiple_files = true;
+option java_package = "io.spoud.sdm.activities.v1";
+option go_package = "activities.v1";
+
+import "google/protobuf/empty.proto";
+import "activities/domain/v1/domain.proto";
+
+service notificationsService {
+  rpc updateNotificationChannelsConfiguration (UpdateNotificationChannelConfiguration) returns (google.protobuf.Empty);
+  rpc getNotificationChannelsConfiguration (google.protobuf.Empty) returns (NotificationChannelsConfigurationResponse);
+}
+
+message UpdateNotificationChannelConfiguration{
+  oneof notificationConfiguration {
+    WebhookConfiguration webhookConfiguration = 1;
+    EmailConfiguaration emailConfiguaration = 2;
+  }
+}
+message WebhookConfiguration{
+  string webhook_url = 1;
+  map<string, string> webhook_headers = 2;
+  repeated io.spoud.sdm.activities.domain.v1.ActivityType.Type notify_only_on_activity_type = 3;
```

Comment:
That could be in UpdateNotificationChannelConfiguration since it's not specific to the "output" of a notification

Reply:
moved to NotificationConfiguration so we can use it in `UpdateNotificationChannelConfiguration` and `NotificationChannelsConfigurationResponse`